content/docs/building-apps/connect-to-anchors/deposit-anchored-assets.mdx

```diff
@@ -259,33 +246,29 @@ The endpoint for this JWT endpoint is noted by the `WEB_AUTH_ENDPOINT` on the TO
 <CodeExample>
 
 ```ts
-...
-
-    const formData = new FormData()
-
-    loEach({
-      asset_code: currency[0],
-      account: this.account.publicKey,
-      lang: 'en'
-    }, (value, key) => formData.append(key, value))
-
-    const interactive = await axios.post(`${this.toml.TRANSFER_SERVER}/transactions/withdraw/interactive`, formData, {
-      headers: {
-        'Authorization': `Bearer ${auth}`,
-        'Content-Type': 'multipart/form-data'
-      }
-    }).then(({data}) => data)
-
-    console.log(interactive)
+const formData = new FormData()
+
+loEach({
+  asset_code: currency[0],
+  account: this.account.publicKey,
+  lang: 'en'
+}, (value, key) => formData.append(key, value))
+
+const interactive = await axios.post(`${this.toml.TRANSFER_SERVER}/transactions/withdraw/interactive`, formData, {
+  headers: {
+    'Authorization': `Bearer ${auth}`,
+    'Content-Type': 'multipart/form-data'
+  }
+}).then(({data}) => data)
 
-...
+console.log(interactive)
 ```
 
 </CodeExample>
 
 This call is our first little bit of magic. Once we setup our `multipart/form-data` entry values we can POST them to the `${this.toml.TRANSFER_SERVER}/transactions/withdraw/interactive` endpoint along with our `auth` Authorization Bearer `auth` token we acquired earlier. The response to this call will contain a special url for interacting directly with the anchor providing the data they need to accept our deposit asset.
 
-<CodeExample>
+<ExampleResponse title="Example Response">
 
 ```json
```

Comment:
Just to confirm - we should use <ExampleResponse/> for json format right? @kylemccollom

Reply:
let's try it and see. that's more for api reference, but could work well in docs, too.